pil-analyzer/src/pil_analyzer.rs

```diff
@@ -542,6 +601,24 @@ impl PILAnalyzer {
     }
 }
 
+impl AllChildren<Expression> for PILAnalyzer {
```

Comment:
If you just implement Children, will you get AllChildren for free?

Reply:
If you implement `Children`, please take care to return the expression itself instead of calling children on them, because in the latter case, it will skip the expression itself.